backend/perf.js

```diff
@@ -16,25 +16,24 @@
 
 // On request for data, produce data in JSON format from our db.
 
-var http = require('http');
-var url = require('url');
-var async = require('async');
-var path = require("path");
-var fs = require('fs');
-var nodegit = require('nodegit');
-var sortedList = require('sortedlist');
+let http = require("http");
+let url = require("url");
+let path = require("path");
+let fs = require("fs");
+let nodegit = require("nodegit");
+let sortedList = require("sortedlist");
 
-var WEEKS_IN_SUMMARY = 12;
+const WEEKS_IN_SUMMARY = 12;
 
 // Data stored by test name and date, each date maps to a table storing the times by crate and by phase.
 // See make_data() for the object layout.
 var data = {};
-var opts = { compare: function(x, y) { return x.date - y.date; }}
-data['rustc'] = sortedList.create(opts);
-data['benchmarks'] = sortedList.create(opts);
+var opts = { compare: function(x, y) { return x.date - y.date; }};
+data.rustc = sortedList.create(opts);
+data.benchmarks = sortedList.create(opts);
 var summary = {};
-summary['rustc'] = [];
-summary['benchmarks'] = [];
+summary.rustc = [];
+summary.benchmarks = [];
```

Comment:
This could be an object literal, rather than setting the fields outside the object